ayon_api/server_api.py

```diff
@@ -370,6 +385,9 @@ def __init__(
         )
         self._sender = sender
 
+        self.timeout = timeout
```

Comment:
Shouldn't we set it to the defaults ( `_default_timeout` and `_default_max_retries` ) `if timeout is None` ?